contracts/externals/ens/RRUtils.sol

```diff
@@ -1,14 +1,12 @@
 pragma solidity >0.4.23;
 
 import "./ENSBytesUtils.sol";
-import "./ENSBuffer.sol";
 
 /**
 * @dev RRUtils is a library that provides utilities for parsing DNS resource records.
 */
 library RRUtils {
     using ENSBytesUtils for *;
-    using ENSBuffer for *;
```

Comment:
Why was thsi removed? even in the case that it is not used maybe it is not a good idea to edit external contracts